landlab/layers/eventlayers.py

```diff
@@ -448,6 +448,11 @@ def event_layers(self):
         finally:
             return self._event_layers
 
+    @property
+    def at_layer(self):
+        """EventLayers for each cell."""
+        return self.event_layers
+
```

Comment:
I *think* this is supposed to be here and not a holdover, but just checking.

Reply:
Yes, I added it so that it would be more like *at_node*, *at_link*, etc.